### DISCUSSION TITLE
How to separate words/phrases in initial_prompt?

### DISCUSSION THREAD

**Discussion by Abdurrafey-Siddiqui** [openai/whisper#594] [Q&A]
So like i want to input separate words. Basically acting like a dictionary. How should I separate the words so that whisper doesn't get confused?

**Comment by dteok** [comment]
i'm interested in this as well, so I'm watching this space

**Comment by Jerrisan** [comment]
Separate the words using **space with double quotation marks** works for me, for example "David James Alice"
This works totally fine for me, but I don't know how to duel with phrases with spaces :(

**Reply by vchagari** [reply]
This does work sometimes, but it is not consistent.
